### DISCUSSION TITLE
slanet测试单元格坐标为空

### DISCUSSION THREAD

**Discussion by zzcqinag** [PaddlePaddle/PaddleOCR#12852] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

用合成数据训练slanet，训练完成之后，测试表格图片，单元格坐标为空。
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/49135781/af00ccc1-cc5e-4d77-8115-21ebf8d60786)

**Comment by crackso** [comment]
啥表格，合并行列都是20？
